app/views/cookies/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= turbo_frame_tag :cookies_modal do %>
   <% if cookies[:cookies_accepted].nil? %>
-    <%= Subdomain.current.cookies_consent_ui.html_safe %>
+      <%= Subdomain.current.cookies_consent_ui.html_safe %>
```

Comment:
@Ayon95 you shouldn't have any changes on this file. You should do 

``` bash
git checkout Pralishcookies-consent -- app/views/cookies/index.html.erb

```

to make sure you reset the file back to how Pralish had it

Reply:
Thanks. I've reset the file.